src/main/java/adris/altoclef/util/ItemTarget.java

```diff
@@ -87,6 +106,12 @@ public int getTargetCount() {
         if (_infinite) {
             return BASICALLY_INFINITY;
         }
+        if (_targetCount == BASICALLY_INFINITY) {
+            _infinite = true;
+            return BASICALLY_INFINITY;
+        }
+        // Workaround for when the ItemTarget is initialized as infinite (properly)
+        // but then another ItemTarget is created by copying the target count, not the infinite flag.
```

Comment:
In that case shouldn't we copy over the `_infinite` flag in the copy constructor/function instead?

Reply:
I think that would require a massive refactoring.

Reply:
So what's the "right" solution here?